commands/import.go

```diff
@@ -212,7 +221,7 @@ func (i *Importer) run() error {
 
 	// print out a message if we're automatically converting this to a rolling database
 	// i.e. it wasn't rolling before and the user didn't specify the --rolling flag
-	if !isRolling && !i.rolling && rollingCfg.Rolling {
+	if exists && !isRolling && rollingCfg.Rolling {
```

Comment:
This would display at an inopportune time. I can't exactly remember what the issue was to be honest.

But this should really only display if we are converting an existing dataset which is what the change reflects.

Reply:
Comment changed.